app/javascript/components/my-studies/StudyUsageInfo.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from 'react'
+import _uniqueId from 'lodash/uniqueId'
+import Plotly from 'plotly.js-dist'
+
+import { fetchStudyUsage } from '~/lib/scp-api'
+import LoadingSpinner from '~/lib/LoadingSpinner'
+import { supportEmailLink } from '~/lib/error-utils'
+import InfoPopup from '~/lib/InfoPopup'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons'
+
+/** display mixpanel stats for a given study */
+export default function StudyUsageInfo({ study }) {
+  const [usageInfo, setUsageInfo] = useState(null)
+  const [isLoading, setIsLoading] = useState(true)
+  const [showTable, setShowTable] = useState(false)
+  const [viewsGraphElementId] = useState(_uniqueId('study-usage-graph-'))
+  const [downloadsGraphElementId] = useState(_uniqueId('study-usage-graph-'))
+  const studyAccession = study.accession
+
+  /** get the usage information via API call */
+  useEffect(() => {
+    fetchStudyUsage(study._id.$oid).then(response => {
+      setUsageInfo(response)
+      setIsLoading(false)
+    })
+  }, [study.accession])
+
+  const tableData = []
+
+  if (usageInfo) {
+    // generate the trace information and table information
+    const viewData = usageInfo.pageViews.data
+    const viewsTrace = {
+      type: 'bar',
+      x: viewData.series,
+      y: viewData.series.map(x => {
+        return viewData.values[x] ? viewData.values[x] : 0
+      })
+    }
+    const viewsLayout = {
+      xaxis: { title: 'Month' },
+      yaxis: { title: 'Unique users' }
+    }
+    Plotly.react(viewsGraphElementId, [viewsTrace], viewsLayout)
+    tableData.push(viewsTrace.x)
+    tableData.push(viewsTrace.y)
+
+    // sum together the single file clicks and the bulk downloads for the graph
+    const downloadData = usageInfo.fileDownloads.data
+    const downloadTableData = []
+    const bulkDownloadData = usageInfo.bulkDownloads.data
+    const bulkDownloadTableData = []
+    const downloadsTrace = {
+      type: 'bar',
+      x: downloadData.series,
```

Comment:
Non-blocking: it would be nice if in the Plotly graph for the hover tootlips `Month` axis, we omit the day.  For instance, instead of saying `March 1, 2022`, it just said `March 2022`.  That might be because we're getting an actual date from Mixpanel for the `Month` axis.  There might be an opportunity to parse that out into a string and omit the day.  Just a thought.

Reply:
Good point -- I'll leave it as is because I think there's some complexity around how partial months get handled.  And we'll definitely keep this in mind if we decide to put some more polish behind this page in the future